pkg/limitador/k8s_objects.go

```diff
@@ -66,6 +66,12 @@ func Deployment(limitador *limitadorv1alpha1.Limitador, deploymentOptions Deploy
 		image = *limitador.Spec.Image
 	}
 
+	templateLabels := Labels(limitador)
+	if limitador.ObjectMeta.Labels != nil {
+		for key, value := range limitador.ObjectMeta.Labels {
+			templateLabels[key] = value
+		}
```

Comment:
I think this allows Limitador CR labels to overwrite the hardcoded labels from `Labels(limitador)` 🤔 Is this the behaviour we want? Or should hardcoded labels take precedence and skip any label that is already hardcoded? 🤔

Reply:
Good point. My take is that the Limitador CR takes precedence. For me, it would be awkward that you set some label in limitador CR and that is not reflected on the deployment. But I am happy to accept the other order of preference.